check/README.md

```diff
@@ -0,0 +1,99 @@
+# Tool for work with string tables
+
+## Glossary
+
+- `string table` a file with translation
+- `reference table` en.txt string table.
+  English is the main game language, so any changes related to text should be in that file.
+  Also this table us used as fallback for other translation.
+  This file is mostly maintained by people who change game code,
+  so the list of entries in this file should be the most actual.
+- `entry` one piece of translation, it consists of
+  - `key` a sting that represents entry in the codebase (C++, Python).
+    This string must be unique.
+  - `value` actual translation that will be shown to the user.
+
+
+### Key
+Only English chars in uppercase and dash are allowed.
+Rename of a key should be done with caution. Usually key used as is in code,
```

Comment:
Rename of a key -> Renaming a key

Reply:
Many of the keys in the stringtable are used in the game source code explicitly, or in content scripts. Some keys are also generated programatically, such as by adding a prefix to other text.

Reply:
Could mention the `UserString("KEY")` or `UserStringNop("KEY")` expressions that appear in C++ code whenever a stringtable key is or could be looked up explicitly.